src/cargo-kani/src/call_goto_instrument.rs

```diff
@@ -28,6 +24,7 @@ impl KaniSession {
         }
 
         if self.args.gen_c {
+            println!("Wrote {}", output.to_string_lossy());
```

Comment:
Is this meant to be user visible? If so, can we expand the message a bit more? Also, should we check if we're in quiet mode?